internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -720,19 +735,34 @@ func (rmq *rocksmq) Consume(topicName string, groupName string, n int) ([]Consum
 		key.Free()
 		offset++
 		msgID, err := strconv.ParseInt(strKey[len(topicName)+1:], 10, 64)
+		fmt.Println("msgid", msgID)
 		if err != nil {
 			val.Free()
 			return nil, err
 		}
+		askedProperties := path.Join(topicName, strconv.FormatInt(msgID, 10), common.PropertiesKey)
```

Comment:
need to refine the properties' key, to avoid read amplification, may be `{topic_name}-properties/{msg_id}`